config/routes.rb

```diff
@@ -0,0 +1,35 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "products#root"
+  resources :products, only: [:index, :show, :new, :create, :edit, :update]
+  resources :order_items, only: [:create, :edit, :update, :destroy]
+
+  resources :products do
```

Comment:
You should only have each `resources` listed once. Lines 4 and 7 are redundant, and line 7 will generate all the routes that line 4 omits!